Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.ts

```diff
@@ -27,6 +54,7 @@ export class AppComponent {
   }
 
   calculateRange() {
-
+	this.range = -(this.speed * this.speed * this.battery / 2500)  +  (4 * this.battery) + this.weather;
+	this.result = "The predicted range of the Eylsia is " + this.range + " km.";
```

Comment:
You can indent the contents of this function with 2 spaces to match your other functions.